src/cargo/ops/cargo_test.rs

```diff
@@ -30,16 +30,19 @@ pub fn run_tests(manifest_path: &Path,
         return Ok(None)
     }
 
-    let libs = compile.package.targets().iter()
```

Comment:
I think this field in `Compilation`, `package`, should probably be removed now as it's not quite as correct as it was before.